express_import/main.ipynb

```diff
@@ -0,0 +1,185 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "55dc7fa8-aa4c-4748-8a2f-c98b030e22fe",
+   "metadata": {},
+   "source": [
+    "## Express Import API Call\n",
+    "This notebook illustrates an example to directly import an S3 asset and add it into a task in a project in one call. This example uses:\n",
+    "\n",
+    "- `requests` library for API calls"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "257a0f4c-ccc4-46d8-acbe-5c6e51e4aad8",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Import required libraries\n",
+    "import requests"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "6a220865-60c8-4b47-afb1-b2a21185ebf3",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# API URLs\n",
+    "auth_url = \"https://api.centaurlabs.com/auth/public/v1/login\"\n",
+    "tasks_url = \"https://api.centaurlabs.com/tasks/public/v1/list?project_id=\"\n",
+    "imports_url = \"https://api.centaurlabs.com/imports/public/v1/add\""
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "95f11d84-2b69-41e2-ba7e-2cf2b9eb803f",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Parameters for headers\n",
+    "email = \"your_portal_email\"  # Replace with your Centaur Email\n",
+    "api_key = \"your_api_key_here\"  # Replace with your Centaur API key\n",
+    "api_password = \"your_api_password_here\"  # Replace with your Centaur API password (not your portal password)\n",
+    "\n",
+    "# Asset parameters\n",
+    "s3_uri = \"s3://s3_uri\" # Replace with your s3 URI\n",
+    "s3_region = \"us-east-2\" # Replace with your S3 Region"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "e1273d34-32c1-492a-8be4-a4c20b89554d",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Function to get the authentication token\n",
+    "def get_auth_token(email, password, key):\n",
+    "    payload = {\"username\": email, \"api_password\": password}\n",
+    "    headers = {\"X-API-KEY\": key}\n",
+    "    response = requests.post(auth_url, headers=headers, json=payload)\n",
+    "    if response.status_code == 200:\n",
+    "        return response.json().get(\"token\")\n",
+    "    else:\n",
+    "        raise Exception(f\"Authentication failed: {response.json()}\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "18c1900d-3f5f-449d-990a-ddd81728f6fa",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Function to list tasks in the project. This can help you decide what tasks to us in the import function\n",
+    "def list_tasks(token, key, project_id):\n",
+    "    headers = {\"Authorization\": f\"Bearer {token}\", \"X-API-KEY\": key}\n",
+    "    url = f\"{tasks_url}{project_id}\"\n",
+    "    print(url)\n",
+    "    response = requests.get(url, headers=headers)\n",
```

Comment:
We can use the [params keyword argument](https://requests.readthedocs.io/en/latest/user/quickstart/#passing-parameters-in-urls) to specify the project_id.